maas/provider.go

```diff
@@ -67,6 +67,7 @@ func Provider() *schema.Provider {
 			"maas_fabric":                     resourceMAASFabric(),
 			"maas_vlan":                       resourceMAASVLAN(),
 			"maas_vlan_dhcp":                  resourceMAASVLANDHCP(),
+			"maas_static_route":               resourceMaasStaticRoute(),
```

Comment:
We're trying to be consistent with acronyms, could you update this please?
`Maas` -> `MAAS`